cmd/cosmos-sdk-cli/cmd/init.go

```diff
@@ -145,8 +146,13 @@ benchmark:
 func setupBasecoinWorkspace(projectName string, remoteProjectPath string) {
 	projectPath := resolveProjectPath(remoteProjectPath)
 	fmt.Println("Configuring your project in " + projectPath)
-	copyBasecoinTemplate(projectName, projectPath, remoteProjectPath)
-	createGopkg(projectPath)
-	createMakefile(projectPath)
-	fmt.Printf("\nInitialized a new project at %s.\nHappy hacking!\n", projectPath)
+	// Check if the projectPath already exists or not
+	if _, err := os.Stat(projectPath); os.IsNotExist(err) {
```

Comment:
Its just the style we've adopted throughout the sdk. I.e.

```
if errCondition {
  error
}
rest of the code
```

It makes it so that the rest of the code isn't wrapped in another pair of curly braces, which you then have to keep track of.

Reply:
You could just use `!os.IsNotExist`, and then print that same error message. That is logically equivalent.